common/src/main/java/io/netty/util/DefaultAttributeMap.java

```diff
@@ -172,6 +172,11 @@ private void remove0() {
                     if (next != null) {
                         next.prev = prev;
                     }
+
+                    // Null out prev and next this will guard against multiple remove0() calls which ma corrupt
```

Comment:
- ma -> my
- Add some punctuation. e.g. Null out prev and next* - *this will guard ...